code/modules/martial_arts/combos/cqc/restrain.dm

```diff
@@ -0,0 +1,22 @@
+/datum/martial_combo/cqc/restrain
+	name = "Restrain"
+	steps = list(MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_GRAB)
+	explaination_text = "Locks opponents into a restraining position, disarm to knock them out with a choke hold."
+	combo_text_override = "Grab, switch hands, Grab"
+
+/datum/martial_combo/cqc/restrain/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	var/datum/martial_art/cqc/CQC = MA
+	if(!istype(CQC))
```

Comment:
I've added it here for safety since the CQC datum is used later on on line 18 and 19. I've tried to make the combos not rely on the martial arts used. But edge cases such this one need to be called by a CQC art.